src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -55,13 +55,18 @@
  * The following example shows how to configure the {@code AbstractClassName} to
  * checks names, but ignore missing {@code abstract} modifiers:
  * </p>
- *
+ * <p>Configuration:</p>
  * <pre>
  * &lt;module name="AbstractClassName"&gt;
  *   &lt;property name="ignoreModifier" value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>Example:</p>
+ * <pre>
+ * abstract class AbstractFirstClass {} // OK
+ * abstract class SecondClass {} // violation, it should match the pattern "^Abstract.+$"
+ * class AbstractThirdClass {} // OK, the missing "abstract" modifier could be ignored
```

Comment:
This check validate only explicitly marked classes with "abstract" modifier to match a format. No modifier = no validation.
please make comment like `OK, no "abstract" modifier`